src/coffee-script.coffee

```diff
@@ -146,13 +163,13 @@ exports.run = (code, options = {}) ->
 
   # Set the filename.
   mainModule.filename = process.argv[1] =
-    if options.filename then fs.realpathSync(options.filename) else '.'
+    if options.filename then fs.realpathSync(options.filename) else '<anonymous>'
```

Comment:
@lydell I changed this so that we have consistent “filenames” when compiling from `run` or `eval` or other non-file sources, which is important for looking up cached sources and source maps later. Do you see any issue with this? Was there anything special about a filename of `.` before?